mission1/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:title" content="판다 마켓" />
+    <meta property="og:description" content="일상의 모든 물건 을 거래해보세요" />
+    <meta property="og:url" content="/" />
+    <link rel="stylesheet" href="style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <div id="wrap">
+      <!-- HEADER -->
+      <div class="header_wrap">
+        <header class="header">
+          <h1 class="logo">
+            <a href="/">판다마켓</a>
+          </h1>
+          <div class="login_box">
+            <a href="login.html" class="login_btn">로그인</a>
+          </div>
+        </header>
+      </div>
+      <!-- // HEADER -->
+
+      <!-- MAIN -->
+      <main>

```

Comment:
💊 제안
주석을 다는 것은 좋은 습관이지만, 가장 좋은 코드는 주석 없이도 이해할 수 있는 코드입니다.  
현재 코드 구조상 주석 없이도 충분히 이해할 수 있어 보이고 추가적인 정보를 전달하지 못하는 것 같아요.
이런 경우 주석이 없는 것을 추천드려요~

```suggestion
      <main>
```